### DISCUSSION TITLE
There are any shortcut for reactive values from child elements?

### DISCUSSION THREAD

**Discussion by dbwodlf3** [lit/lit#3396] [General]
When make some components, in many case i have to syncro values from elements.

```ts
@proerty()
value = ""

@query("#ExampleInput")
exampleInputElement;

render(){
    return `<input id="ExampleInput" type="text" value=${this.value}>`
}
```

ExampleInput's value is changed but not changed components value.
so i have to add input event and syncro values.
it is trivit but intrusive and annoy me.

there are any good idea? or just i have to keep doing this..?

**Comment by augustjk** [comment]
Lit offers uni-directional data flow or one-way data binding. React works like this too.

What you're asking for is two-way data binding like in Angular. It's not a simple thing to add that kind of behavior to Lit. You could potentially write a directive that helps you. There are some more discussion and ideas here https://github.com/lit/lit/issues/253.
